sachungo/movies
# Filter actors

### Which issue does this PR relate to?

https://www.pivotaltracker.com/story/show/159732917

### What are the changes implemented in the PR?



- Added the ability to filter movies by actors
- Fixed some bugs in the paginator logic
- Added tags that show up on selecting a filter criteria

### How should the implementation be QA'd?



- Visit the homepage of the deploy - the link is available below
- Clicking `Actors` button once the button is enabled and filter movies. Verify that on selecting an item, 
    - the `Clear` button is shown
    - clicking the `Clear` button does not close the dropdown when a filter criteria hasn't yet been applied
    - Clicking the `Clear` button closes the dropdown when a filtering criteria had been applied before
    - Clicking the `Apply` button applies the filtering criteria and closes the dropdown
- Verify that the pagination works with the filtered results
- Verify that on mobile, clicking the filter button opens a modal


### Anything else to note


N/A

### Screenshots and/or gifs


**Mobile**
![screen recording 2018-08-31 at 10 11 am](https://user-images.githubusercontent.com/12892171/44898128-a259c000-ad06-11e8-9d06-8bc24b4077d6.gif)

**Desktop**
![screen recording 2018-08-31 at 10 14 am](https://user-images.githubusercontent.com/12892171/44898146-b2719f80-ad06-11e8-98f1-b8b128329305.gif)
